data/data/aws/master/outputs.tf

```diff
@@ -1,7 +1,3 @@
-output "subnet_ids" {
-  value = "${var.subnet_ids}"
-}
-
 output "cluster_id" {
   value = "${var.cluster_id}"
```

Comment:
This is, like `subnet_ids` above, also a useless "tell them what they told you" output.  Looks like we've been dragging them around since e5c8b41 (platform/aws: add bootstrap node and step for joining it, 2018-02-13, coreos/tectonic-installer#2924).